helpers/middleware/logger.ts

```diff
@@ -37,29 +34,16 @@ export const winstonLogger = (sessionId: string) => {
       winston.format.colorize(),
       winston.format.printf(printFunc)
     ),
-    transports: [
-      new winston.transports.Console(),
-      new Sentry({
-        sentry: {
-          dsn: SENTRY_DSN
-        }
-      })
-    ],
-    exceptionHandlers: [
-      new winston.transports.Console(),
-      new Sentry({
-        sentry: {
-          dsn: SENTRY_DSN
-        }
-      })
-    ]
+    transports: [new winston.transports.Console()]
   })
 }
 
 export default (req: LoggedRequest, res: NextApiResponse, next: () => void) => {
   const uid = nanoid()
   const logger = winstonLogger(uid)
+
   req.info = (val: any) => {
+    console.log('capture info')
```

Comment:
Did you forget to remove this console.log?

Reply:
I did, ty